src/styles/views/Lobby.scss

```diff
@@ -0,0 +1,106 @@
+@import "../theme";
+
+/* The '&' is a shorthand for the containing element,
+  so for example a '&.user-list' inside .game will compile to
+  .game.user-list
+
+  for more information visit https://sass-lang.com/guide
+ */
+
+.lobbypage {
+
+  &.container {
+    float: left;
+    background: $background;
+    padding: 1.5em;
+    border-radius: $borderRadius;
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    box-shadow: $dropShadow;
+    margin-left: 30px;
+  }
+
+  & p,
+  h2 {
+    margin: 0.5em 0;
+  }
+}
+
+.gamemode {
+
+  &:hover {transform: translateY(-2px);}
+  &.container {
+    text-align: start;
+    margin: 0.5em 0;
+    width: auto;
+    padding: 20px;
+    border-radius: $borderRadius;
+    display: flex;
+    align-items: center;
+    background: lighten($background, 20);
+    cursor: pointer;
+
+    &.selected {
+      background: lighten($background, 25);
+      border: solid rgb(0, 0, 0)
+    }
+  }
+
+    &.name {
+    text-align: center;
+    margin: auto;
+    color: $accentDark;
+    text-transform: uppercase;
+  }
+
+  &.description {
+    color: $textColor;
+
+  }
+
+  &.list {
+    list-style: none;
+    text-align: center;
+  }
+
+}
+
+.player {
+
+  &.container {
+    float: left;
```

Comment:
Remove `float:left`
Change `padding: 20px` to `padding: 4px`
Change `flex-direction: column` to `flex-direction: row` (will display player names to the right of icon)